packages/js-client-grpc/src/api-client.ts

```diff
@@ -49,7 +50,21 @@ function createClients(transport: Transport) {
     } satisfies Clients;
 }
 
-export function createApis(baseUrl: string, {timeout, apiKey}: {timeout: number; apiKey?: string}): GrpcClients {
+type CreateApisParams = {
+    timeout: number;
+    apiKey?: string;
+    useCompression: boolean;
```

Comment:
Can `useCompression` parameter be undefined? I mean 
`useCompression: boolean | null | undefined;`